include/selene/State.h

```diff
@@ -18,6 +18,39 @@ class State {
     std::unique_ptr<Registry> _registry;
     std::unique_ptr<ExceptionHandler> _exception_handler;
 
+    /// Shared code post-calling LoadString or LoadFile
+    inline bool PostLoad(int status, const char* file = nullptr) {
```

Comment:
On the case of LoadStr , simply isn't used. file string it's only to show the filename that failed. If we are on LoadStr, there isn't a filename to print, so it'¡s set to null.